feat(config)!: force successful SSG generation,  remove `ssg.strict` and add `ssg.fallback` (#2015)
web-infra-dev/rspress: packages/document/docs/en/api/config/config-basic.mdx

## ssg

- Type: `boolean | { strict?: boolean }`
- Default: `{ strict: true }`

> Whether to enable static site generation, Rspress is enabled by default to generate CSR outputs and SSG outputs.

Reviewer: ```suggestion
Whether to enable static site generation. Rspress enable it by default to generate CSR outputs and SSG outputs.
```
